src/renderer/components/ft-share-button/ft-share-button.scss

```
@@ -79,13 +81,15 @@
         .dracula &,
         .CatppuccinMocha &,
         .hotPink &,
+	.gruvboxDark &,
         .solarizedDark &,
         .system[data-system-theme*='dark'] & {
           background-image: url('../../assets/img/invidious-logo-dark.svg');
         }
 
         .pastelPink &,
         .light &,
+	.gruvboxLight &,
```

```suggestion
        .light &,
```

Remove extraneous reference to gruvBoxLight